versions/v060/rn-knative-v060-OCP-4x.md

```diff
@@ -16,37 +16,50 @@ This document describes the latest changes, enhancements, bug fixes and known is
 
+- **Operator automatically installs the Maistra Operator and ControlPlane.** The operator will attempt to discover Istio. If not found, it will install the Maistra Operator and a minimal Maistra ControlPlane. ([#12](https://github.com/openshift-knative/knative-serving-operator/pull/12))
+
+- **Operator automatically installs the Knative OpenShift Ingress Operator.** After installing Knative Serving, the operator now ensures that the Knative OpenShift Ingress Operator is deployed so that OpenShift Routes get created automatically for every Knative Route. ([#14](https://github.com/openshift-knative/knative-serving-operator/pull/14))
 
 ### Eventing
-- **Brief:** Description. [(#PR)](link to PR)
 
-### Build
-- **Brief:** Description. [(#PR)](link to PR)
+- **Knative Eventing Operator now available.** An Operator that installs the Knative Eventing component on OpenShift 4.1 is now available through the embedded OperatorHub in the  OpenShift console. This replaces the previous installation script available in Knative on OpenShift v0.5.0. ([#407](https://github.com/operator-framework/community-operators/pull/407))
 
 -------------
 
 ## Fixed Issues
 
+### Serving
+- See the [Knative core Serving release notes](https://github.com/knative/serving/releases).
+
 ### Eventing
-- Description of fix. [(#PR)](link to PR)
+- See the [Knative core Eventing release notes](https://github.com/knative/eventing/releases).
 
-### Build
-- Description of fix. [(#PR)](link to PR)
 -------------
 
 ## Known Issues
-- **Brief:** Description. [(#PR)](link to PR)
+- **The Maistra ControlPlane remains after Knative Serving has been removed.** The Maistra control plane installed by the Knative Serving Operator when Istio is not found does not get removed when Knative Serving is uninstalled. ([#12](https://github.com/openshift-knative/knative-serving-operator/pull/12))
+
+- **Camel and Kafka optional eventing sources have been removed.** The Camel and Kafka sources that were previously available by installing the Knative Eventing Operator have been removed. These will become standalone Knative Kafka and Knative Camel Operators in a future release of Knative on OpenShift. See *Resources and Links* below for more information. ([#34](https://github.com/openshift-knative/knative-eventing-operator/pull/34), [#1](https://github.com/openshift-knative/knative-kafka-operator/pull/1))
+
+- **Activator and autoscaler pods fail to start when there is an existing Maistra installation before installing the Knative Serving Operator.** If Maistra version 0.10 or 0.11 is already installed before installing the Knative Serving Operator, the activativator and autoscaler pods can fail to start, and the user will encounter an error. This issue can temporarily be fixed by setting the Security Context Constraints for service accounts controller and default to `privileged`;
```

Comment:
activativator -> activator :)

Reply:
Actually, I think this only applies to Maistra version 0.10. Please remove "or 0.11" from the text. Sorry for not catching earlier.

Reply:
No worries, thanks!